src/plots/mapbox/mapbox.js

```diff
@@ -351,10 +353,12 @@ proto.updateLayout = function(fullLayout) {
     var map = this.map;
     var opts = fullLayout[this.id];
 
-    map.setCenter(convertCenter(opts.center));
-    map.setZoom(opts.zoom);
-    map.setBearing(opts.bearing);
-    map.setPitch(opts.pitch);
+    if(!this.dragging && !this.wheeling) {
+        map.setCenter(convertCenter(opts.center));
+        map.setZoom(opts.zoom);
+        map.setBearing(opts.bearing);
+        map.setPitch(opts.pitch);
+    }
```

Comment:
Very interesting solution. I think this should work. Would you be interested in trying to write a few tests to lock this down?

Reply:
Tests added in https://github.com/plotly/plotly.js/pull/4418/commits/4f0c5e7a526b3c9c4b800b4d301f2377b3203e9b